src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -0,0 +1,77 @@
+; Define the set of instruction sets available on a platform
+; Format is 
+;
+; Add new instruction set
+; instructionset,<architecture>,<managed name>,<r2r name if different>,<R2R numeric value>, <jit instruction set name>
+;
+; Add jit 64bit architecture specific instruction set when instruction set is available
+; instructionset64bit,<architecture>,<jit instruction set name>
+;
+; Add an instruction set implication (i.e, if instruction set A is present, then instruction set B must be present too.)
+; implication,<architecture>,<jit instruction set name>,<implied jit instruction set name>
+;
+; Copy instruction sets defined for other architecture at this point in the file.
+; copyinstructionsets,<architecture>,<copytoarchitecture>
+
+; Definition of X86 instruction sets
+
+instructionset,X86,Sse,,1,SSE
```

Comment:
This would be easier to read (and hopefully not significantly harder for the tool to parse) if the columns lined up.

Reply:
Done